apps/web/src/hooks/use-export-collectible.ts

```diff
@@ -26,6 +24,12 @@ export function useExportCollectible(passphrase: string) {
   const [selectedAccount, selectAccount] = useState('')
   const [exportStatus, setExportStatus] = useState<ExportStatus>('idle')
   const connectorReference = useRef<IConnector>()
+  const Environment = useConfig()
```

Comment:
Might it make sense to name this 'Config' or something other than 'Environment' since it's on the client side? Just to avoid confusion with the server>client environment variables?

Reply:
yeah you're totally right, I guess I was being lazy about updating these